scripts/ci/lib.sh

```diff
@@ -1409,6 +1414,26 @@ EOT
     hub-comment -type build -template-file "$tmpfile"
 }
 
+get_junit_parse_cli() {
+    # TODO !! Need to remove this commit hash before merging
+    go install github.com/stackrox/junit-parse@blugo/4e2c784
+}
+
+junit2slack() {
+    info "Converting junit failures to slack attachments"
+
+    if [[ -z "${ARTIFACT_DIR}" ]]; then
+        info "Warning: junit2slack() requires an ARTIFACT_DIR"
+        return
+    fi
+
+    local junit_file_names="$ARTIFACT_DIR/**.*.xml"
+    pushd "$SCRIPTS_ROOT/tools/junit2slack" || return
```

Comment:
Is this needed anymore? And I expect the dir is no longer present now that you have moved the go code.